src/bin/bat/directories.rs

```diff
@@ -9,6 +9,7 @@ use lazy_static::lazy_static;
 pub struct BatProjectDirs {
     cache_dir: PathBuf,
     config_dir: PathBuf,
+    system_config_dir: Option<PathBuf>,
```

Comment:
Thanks for taking the time to make a contribution!

I understand this is temporary, but making a comment so we don't forget:

There shall always be a system_config_dir, so we should make it a non-`Option`